feat: Implements functionality for saving test logs, closes #46

gustajoh opened the pull request:
@JacobTrossing I added test logs to the GitHubSchema object in Flask_application.py on line 77 and to the construct mail function, however I am unsure if it needs to be added somewhere else to get it working fully

gustajoh committed:
feat: Implements functionality for saving test logs, closes #46

gustajoh committed:
fix: fixed unit tests to work again with the update tuple from initialization, closes #48

JacobTrossing commented:
Think a test_logs field is required in the GithubSchema. After merging with main it needs to show the logs in the build/<id> page recently added.

JacobTrossing committed:
feat: #46 Merges with main and displays test logs in the build page

Also adds a test logs field to GithubSchema.
Merges remote-tracking branch 'origin/main' into test_logs.
